frontend/src/component/personalDashboard/PersonalDashboard.tsx

```diff
@@ -187,9 +212,14 @@ export const PersonalDashboard = () => {
                     id='flags-panel-header'
                     aria-controls='flags-panel-content'
                 >
-                    <Typography variant='body1' component='h3'>
-                        My feature flags
-                    </Typography>
+                    <AccordionSummaryText>
+                        <AccordionSummaryHeader>
+                            My feature flags
+                        </AccordionSummaryHeader>
+                        <AccordionSummarySubtitle>
+                            Feature flags you have created or favorited
```

Comment:
in the projects screen we end the sentence with a dot, but here there's no dot. We should update one or the other. I'd vote for no dot

Reply:
Deal.